source/core/parameters.cc

```diff
@@ -291,6 +291,62 @@ namespace Parameters
     prm.leave_subsection();
   }
 
+  void
+  InterfaceSharpening::declare_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("interface sharpening");
+    {
+      prm.declare_entry("sharpening threshold",
+                        "0.5",
+                        Patterns::Double(),
+                        "VOF interface sharpening threshold");
+      // This parameter must be larger than 1 for interface sharpening. Choosing
+      // values less than 1 leads to interface smoothing instead of sharpening.
+      prm.declare_entry("interface sharpness",
+                        "2",
+                        Patterns::Double(),
+                        "VOF interface sharpness");
```

Comment:
add a better description of the interface sharpness here instead of just "VOF interface sharpness"

Reply:
Added.